tests/bdd/features/rebuild/test_log_based_rebuild.py

```diff
@@ -42,19 +40,30 @@
 POOL_2_UUID = "91a60318-bcfe-4e36-92cb-ddc7abf212ea"
 POOL_3_UUID = "4d471e62-ca17-44d1-a6d3-8820f6156c1a"
 POOL_4_UUID = "d5c5e3de-d77b-11ed-afa1-0242ac120002"
-RECONCILE_PERIOD_SECS = 1
-FIO_RUNTIME = 120
+VOLUME_SIZE = 524288000
+POOL_SIZE = 734003200
+NUM_VOLUME_REPLICAS = 3
+RECONCILE_PERIOD_SECS = "3s"
+FAULTED_CHILD_WAIT = "12s"
+FIO_RUN = 15
+SLEEP_BEFORE_START = 5
+WAIT_FOR_FULL_REBUILD = 16
+REPUBLISH_WAIT = 12
 faulted_child_uri = None
+nexus_local_child = None
+new_target_node = None
```

Comment:
what are these for?

Reply:
there is one more case im working on. Its used in that case